components/brave_today/rust/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "brave-news-cxx"
+version = "1.0.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cxx = { path = "../../../../third_party/rust/cxx/v1/crate"}
+env_logger = "0.9.0"
```

Comment:
This doesn't appear to be used. It's only referred to in a comment. Should we take it out?

Reply:
Or alternatively, if it is useful for debug builds we can conditionally include it with a [feature](https://doc.rust-lang.org/cargo/reference/features.html).

Reply:
I'd prefer we just remove the dep. This is a library crate; the logging implementation is the caller's job.

I guess it's a little weird if your only caller is C++. The clean thing might be to provide your own logger implementation that calls over cxx-bridge to `VLOG`.

Reply:
removed